### PR TITLE
Update to latest PHPUnit 10 syntax

### PR DESCRIPTION
**Pull request by samsonasik**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Updated to latest PHPUnit 10 syntax version based on defined in composer.json

### CODE REVIEW

**Comment by samsonasik** [comment]
Ready for review 👍

File: test/EventTest.php [hunk 1]
```diff
@@ -6,12 +6,11 @@
...
 use Laminas\EventManager\Event;
 use Laminas\EventManager\Exception;
+use PHPUnit\Framework\Attributes\Group;
 use PHPUnit\Framework\TestCase;
 use stdClass;
 
-/**
- * @group      Laminas_Stdlib
- */
+#[Group('Laminas_Stdlib')]
```
Comment (gsteel) [inline test/EventTest.php]: Can we grop the Group annotations completely - they're just noise

File: test/FilterIteratorTest.php [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
 use Laminas\EventManager\Exception\InvalidArgumentException;
 use Laminas\EventManager\Filter\FilterIterator;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Group;
 use PHPUnit\Framework\TestCase;
 
-/**
- * @group      Laminas_Stdlib
- */
+#[Group('Laminas_Stdlib')]
```
Comment (gsteel) [inline test/FilterIteratorTest.php]: Same here

**Review by gsteel — CHANGES_REQUESTED** [review]

File: test/EventTest.php (hunk 1 above)
Comment (samsonasik) [reply test/EventTest.php]: Removed 👍 https://github.com/laminas/laminas-eventmanager/pull/61/commits/386056eff695fc72c6e7545d95d5a3fd9e9df148

File: test/FilterIteratorTest.php (hunk 2 above)
Comment (samsonasik) [reply test/FilterIteratorTest.php]: Removed 👍 https://github.com/laminas/laminas-eventmanager/pull/61/commits/386056eff695fc72c6e7545d95d5a3fd9e9df148

**Review by gsteel — APPROVED** [review]
Thanks @samsonasik 👍

**Pull request merged**